hazelcast-sql/src/test/java/com/hazelcast/sql/SqlErrorClientTest.java

```diff
@@ -149,29 +118,12 @@ public void testMemberDisconnect_execute() {
         instance1 = newHazelcastInstance(true);
         client = newClient();
 
-        populate(instance1);
+        SqlStatement streamingQuery = new SqlStatement("SELECT * FROM TABLE(GENERATE_STREAM(100_000))");
 
-        BlockingExec.Blocker blocker = new BlockingExec.Blocker();
+        Thread thread = new Thread(() -> instance1.shutdown());
```

Comment:
Should not we start query first, then shutdown a member?

Reply:
`Thread.sleep()` does not guarantee any ordering, we should start the query first then having that shutdown a member.